pliers/extractors/api/awsReko_test.py

```diff
@@ -0,0 +1,82 @@
+'''
+Extractors that interact with the AWS Rekognition API.
+'''
+
+from pliers.stimuli.image import ImageStim
+from pliers.extractors.base import Extractor, ExtractorResult
+import numpy as np
+import pandas as pd
+import boto3
+from pliers.utils import attempt_to_import, verify_dependencies
+
+aws_rekognition_client= attempt_to_import('boto3')
+
+
+class AwsRekognitionExtractor():
+    
+    def __init__(self,profile_name='default', region_name='us-east-1' ):
+            verify_dependencies(['boto3'])
+            self.session = boto3.Session(profile_name='user2')
+            self.rekognition = boto3.Session.client('rekognition', region_name='us-east-1')
+            super(AwsRekognitionExtractor, self).__init__()
+    
+    
+    def _extract(self, stim):
+        img2byte=self.img_to_byte(stim)
+        response=self.rekognition.detect_faces(
+            Image={
+                'Bytes': img2byte
+                },
+                    
+            Attributes=['ALL']
+        )
+        return response
+    
+    
+    @property
+    def img_to_byte(img):
```

Comment:
This will work for now, but I have an open PR I'm hoping to merge in the next couple of days that adds this functionality to the `Stim` classes directly. Once it's merged, you'll be able to do `data  = img.get_bytestring()` instead of having to include and call the custom `img_to_byte` method.